raaghu-elements/src/rds-slider/rds-slider.tsx

```
@@ -40,46 +40,37 @@
-  rightLabel = "100", // Default label for the right side
-  level = 1,
-  style = "default", // Default style
-  value: propValue = 0, // Default value prop
-  onChange, // Callback to handle value changes
-}) => {
+const RdsSlider= (props:RdsSliderProps) =>{
+  const propValue = props.value || 0; // Default value prop
   const [value, setValue] = useState(propValue);
   const [rangeValues, setRangeValues] = useState<[number, number]>([0, 100]);
-
+    const [isTouched, setIsTouched] = useState(false);
```

ESLint

[error] 'setIsTouched' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)